lib/pages/forms_pages/adult_report_page.dart

```diff
@@ -75,7 +75,7 @@ class _AdultReportPageState extends State<AdultReportPage> {
     _pagesController = PageController();
     index = 0.0;
     _initialformsRepot = [
-      AddPhotoButton(true, true, _checkAtLeastOnePhotoAttached),
+      AddPhotoButton(true, true, _checkAtLeastOnePhotoAttached, 'ensure_single_mosquito_photos'),
```

Comment:
Should here be using the already translated string instead of the string id? I mean directly the `MyLocalizations.of(context, 'ensure_single_mosquito_photos')`

Reply:
I tried it like that at first, but then get an exception
![image](https://github.com/Mosquito-Alert/Mosquito-Alert-Mobile-App/assets/157690872/7d924136-8ff8-456e-99be-ebdec269d867)

Reply:
so then, maybe it make more sense to just simply rename subtitle to subtitled or similar

Reply:
maybe `subtitleKey` makes sense since it's the key to be translated inside the `AddPhotoButton` widget